<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plan</class>
 <widget class="QWidget" name="Plan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_info">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>flechazo</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget_info"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_info">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;CCTools&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ÊàëÂú®‰∫∫Èó¥Ë¥©ÂçñÈªÑÊòè‚ú®&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Âè™‰∏∫Êî∂ÈõÜ‰∏ñÈó¥Ê∏©ÊüîüíåÈÅáËßÅ‰Ω†&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Êó®Âú®ÊàëÁöÑÊÉ≥Ê≥ï„ÄÅËßÑÂàí‰∏çÁÉÇÂ∞æüíÆ&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_info">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_plan">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_pursuit">
      <attribute name="title">
       <string>‰∫∫Áîü‰∫ã‰∏ö</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_pursuit_operate">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="title">
          <string>ÈÖçÁΩÆ</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_pursuit_operate_path">
            <property name="title">
             <string>Ë∑ØÂæÑ</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_pursuit_operate_path">
               <property name="minimumSize">
                <size>
                 <width>360</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_path">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>ÈÄâÊã©mdÊñá‰ª∂Ë∑ØÂæÑ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_flush">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>ËØªÂèñÂà∑Êñ∞</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_pursuit_operate_collect">
            <property name="title">
             <string>Êìç‰Ωú</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_new">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>Êñ∞Âª∫</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_edit">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>ÁºñËæë</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_showdetail">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Ê∏≤Êüì</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pursuit_operate_delete">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>Âà†Èô§</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_pursuit_view">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_pursuit_view_table">
          <attribute name="title">
           <string>Ë°®Ê†ºËßÜÂõæ</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTreeWidget" name="treeWidget_pursuit_view_table_overview">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_pursuit_view_table_detail" native="true">
             <property name="minimumSize">
              <size>
               <width>360</width>
               <height>180</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_pursuit_view_xmind">
          <attribute name="title">
           <string>ÊÄùÁª¥ÂØºÂõæ</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_pursuit_view_timeline">
          <attribute name="title">
           <string>Êó∂Èó¥Á∫ø</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_study">
      <attribute name="title">
       <string>ÁªàË∫´Â≠¶‰π†</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_life">
      <attribute name="title">
       <string>ËÆ§ÁúüÁîüÊ¥ª</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_love">
      <attribute name="title">
       <string>ËßÑÂàíÁà±ÊÉÖ</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_family">
      <attribute name="title">
       <string>ÂÆ∂Â∫≠ËÆ°Âàí</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_work">
      <attribute name="title">
       <string>Â∑•‰Ωú</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
